### DISCUSSION TITLE
Can we remove or lessen the size of the dots that appear on the chart while seeing the tooltip?

### DISCUSSION THREAD

**Discussion by inder504** [imaNNeo/fl_chart#1300] [Q&A]
![sleep_graph](https://user-images.githubusercontent.com/86961700/230032521-7dc709a5-dbb3-4489-af4e-0b205226ae80.jpeg)

**Comment by imaNNeo** [comment]
If you handle the showing tooltip yourself, then you can provide new data for showing dots (with new size) when showing tooltip.
